src/deserialize/parse.rs

```diff
@@ -431,6 +459,18 @@ mod test {
         )
     }
 
+    #[test]
+    fn parse_str_with_escaped_characters() {
+        let mut string = r##""hello\n \r \t \"world\" with escaped \\ \g characters""##
```

Comment:
Does this make sense to allow? I would throw an error saying that the String has an unknown character `\g`.

```
_ => {
    // On an unsupported escape character, output both the backslash and the character
   s.push('\\');
   s.push(c.1);
}
```

Would raise something like `Error::ParseEdnString("Could not parse string char {}.to_string())` where `{}` would be `\g`. What do you think?